packages/dashboard/tests/frontend_selenium/tests/test_farm.py

```diff
@@ -161,16 +162,16 @@ def test_farmpayout_address(browser):
     polka_page.authenticate_with_pass(password)
     assert farm_page.wait_for('Transaction submitted')
     assert farm_page.wait_for('Address added!')
-    assert farm_page.wait_for('Edit')
-    assert farm_page.wait_for('GDHJP6TF3UXYXTNEZ2P36J5FH7W4BJJQ4AYYAXC66I2Q2AH5B6O6BCFG')
+    assert farm_page.farmpayout_address_value() == 'GDHJP6TF3UXYXTNEZ2P36J5FH7W4BJJQ4AYYAXC66I2Q2AH5B6O6BCFG'
     farm_page.close_detail()
     case = "GA2CWNBUHX7NZ3B5GR4I23FMU7VY5RPA77IUJTIXTTTGKYSKDSV6LUA4"
     farm_page.setup_farmpayout_address(farm_name)
+    browser.find_element(*farm_page.edit_stellar_address).click()
     farm_page.add_farmpayout_address(case).click()
     polka_page.authenticate_with_pass(password)
     assert farm_page.wait_for('Transaction submitted')
     assert farm_page.wait_for('Address added!')
-    assert farm_page.wait_for('GA2CWNBUHX7NZ3B5GR4I23FMU7VY5RPA77IUJTIXTTTGKYSKDSV6LUA4')
+    assert farm_page.farmpayout_address_value() == 'GA2CWNBUHX7NZ3B5GR4I23FMU7VY5RPA77IUJTIXTTTGKYSKDSV6LUA4'
```

Comment:
The test failed 4 out of 4 times at this assertion with the following error

```
E       AssertionError: assert 'GDHJP6TF3UXY...Q2AH5B6O6BCFG' == 'GA2CWNBUHX7N...GKYSKDSV6LUA4'
E         - GA2CWNBUHX7NZ3B5GR4I23FMU7VY5RPA77IUJTIXTTTGKYSKDSV6LUA4
E         + GDHJP6TF3UXYXTNEZ2P36J5FH7W4BJJQ4AYYAXC66I2Q2AH5B6O6BCFG

tests/test_farm.py:174: AssertionError
```
i think it's missing a wait before the assertion to give the dashboard a chance to load the new address instead of asserting for it right after it's applied.